README.md

```diff
@@ -54,4 +54,6 @@ Examples of the points (rename `tags.json.example` to `tags.json` for successful
 ## Methods
 * POST `/api/points` with point in the body adds a new point to the storage, validating it against corresponding tag
 * GET `/api/points` returns the entire storage contents
-* PUT `/api/points` overwrites the entire storage, the body format is the same as result of the previous method; all points are validated against tags, if any single one doesn't pass validation, storage is not modified
\ No newline at end of file
+* GET `/api/points?startDate=yyyy-mm-dd` returns all points that were added on or after specified date
```

Comment:
nit:
"added on or after"?
returns all points that were added on < yyyy-mm-dd > or after < yyyy-mm-dd >